experiments/cassandra/pod-delete/pod_delete_ansible_logic.yml

```diff
@@ -0,0 +1,114 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars:
+    c_experiment: "cassandra-pod-delete"
+    c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+    c_interval: "{{ lookup('env','CHAOS_INTERVAL') }}"
+    ramp_time: "{{ lookup('env','RAMP_TIME') }}"
+    c_force: "{{ lookup('env','FORCE') }}"
+    c_lib: "{{ lookup('env','LIB') }}"
+    c_svc_acc: "{{ lookup('env','CHAOS_SERVICE_ACCOUNT') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    a_kind: "{{ lookup('env','APP_KIND') }}"
+    kill_count: "{{ lookup('env','KILL_COUNT') }}"
+    chaos_uid: "{{ lookup('env','CHAOS_UID') }}"
+    c_engine: "{{ lookup('env','CHAOSENGINE') }}"
+    chaos_pod_name: "{{ lookup('env','POD_NAME') }}"
+  
+  tasks:
+    - block:
+
+        - include: pod_delete_ansible_prerequisites.yml
+      
+        - include_vars:
+            file: chaosutil.yml
+
```

Comment:
need to write comments on these tasks.
can you write full Experiment in place of EXP in line 29